AIPscan/Reporter/report_aip_contents.py

```diff
@@ -1,37 +1,102 @@
 # -*- coding: utf-8 -*-
+from operator import itemgetter
 
 from flask import render_template, request
 
+from AIPscan import db
 from AIPscan.Data import data, fields
-from AIPscan.helpers import get_human_readable_file_size
-from AIPscan.Reporter import reporter, request_params, translate_headers
+from AIPscan.helpers import filesizeformat, parse_bool
+from AIPscan.models import AIP, File, FileType, StorageService
+from AIPscan.Reporter import download_csv, reporter, request_params, translate_headers
+
+HEADERS = [
+    fields.FIELD_UUID,
+    fields.FIELD_AIP_NAME,
+    fields.FIELD_CREATED_DATE,
+    fields.FIELD_AIP_SIZE,
+]
+
+
+def _prepare_csv_data(storage_service_id, rows, puids):
```

Comment:
Codecov is rightly interested in this. I guess if it doesn't work during the user testing phase, it'll be pretty obvious when it fails. But is there any kind of test coverage we can implement here to do checking around this function?